io.openems.edge.evcs.cluster/readme.md

```diff
@@ -0,0 +1,8 @@
+# io.openems.edge.controller.evcs.collection Provider
+
+${Bundle-Description}
+
+## Example
+
+## References
+
```

Comment:
Replace with readme.md file. Always provide documentation for your bundles - otherwise it's unusable